src/molecules/Select/Select.styles.ts

```diff
@@ -51,8 +51,12 @@ const Container = styled.div<ContainerProps>`
   grid-template-columns: 1fr auto;
   align-items: center;
   padding: 6px 8px;
+  transition:
+    background ${animation.transitionMS},
+    box-shadow ${animation.transitionMS};
```

Comment:
It seems like this box-shadow transition is not needed, as you only set cursor and background on hover 🤔

Reply:
The box shadow hover is set in the different variants below, just setting the transition here so we can transition both properties